src/index.js

```diff
@@ -0,0 +1,65 @@
+import './style.css';
+import {
+  addItem,
+  removeItem,
+  displayToDos,
+  getDescriptionInput,
+  markCompleted,
+  clearMethod,
+} from './CRUD.js';
+
+const todoContainer = document.querySelector('.todos-container');
+const addBtn = document.querySelector('.fa-plus');
+const input = document.querySelector('.toDoName');
+const locStorage = JSON.parse(localStorage.getItem('todos'));
+const clearCompleted = document.querySelector('.clear-completed');
+
+let arr;
+if (locStorage == null) {
+  arr = [];
+} else {
+  arr = locStorage;
+}
+
+window.addEventListener('load', () => {
+  displayToDos(arr, todoContainer);
+});
+
+addBtn.addEventListener('click', () => {
```

Comment:
- [ ] Consider using `submit` event instead of click as u are using a `form`

Reply:
Thanks for reaching out to me and giving me thorough feedback. I have implemented all the changes which you have suggested to me. I am hoping that in the future, you will collaborate with me again Inshallah.